src/gui/titlebar.cpp

```diff
@@ -0,0 +1,878 @@
+			config.version = TitlebarConfig::VersionDisplay::Detailed;
+			continue;
+		}
+
+		if (iequals(setting, "cycles")) {
+			maybe_warn_double_cycles();
+			config.show_cycles = true;
+			continue;
+		}
+
+		if (iequals(setting, "mouse")) {
+			maybe_warn_double_mouse();
+			config.mouse = TitlebarConfig::MouseHintDisplay::Default;
+			continue;
+		}
+
+		if (iequals(setting, "mouse=none")) {
+			maybe_warn_double_mouse();
+			config.mouse = TitlebarConfig::MouseHintDisplay::None;
+			continue;
+		}
+
+		if (iequals(setting, "mouse=short")) {
+			maybe_warn_double_mouse();
+			config.mouse = TitlebarConfig::MouseHintDisplay::Short;
+			continue;
+		}
+
+		if (iequals(setting, "mouse=full")) {
+			maybe_warn_double_mouse();
+			config.mouse = TitlebarConfig::MouseHintDisplay::Full;
+			continue;
+		}
+
+		LOG_WARNING("SDL: Invalid 'window_titlebar' setting '%s', ignoring",
+		            setting.c_str());
+		config_needs_sync = true;
+	}
+
+	if (config_needs_sync) {
+		sync_config();
+	}
+}
+
+void TITLEBAR_ReadConfig(const Section_prop& secprop)
+{
+	parse_config(secprop.Get_string("window_titlebar"));
+
+	GFX_RefreshTitle();
+}
+
+void TITLEBAR_AddConfig(Section_prop& secprop)
+{
+	constexpr auto always = Property::Changeable::Always;
+
+	Prop_string* prop_str = nullptr;
+
+	prop_str = secprop.Add_string("window_titlebar",
+	                              always,
+	                              "program=name dosbox cycles mouse=full");
+	prop_str->Set_help(
+	        "Space separated list of information to be displayed on the window title bar\n"
+	        "('program=name dosbox cycles mouse=full' by default).\n"
+	        "Possible information to display are:\n"
+	        "  animation=off:      Disable animated audio/video recording mark. Use if your screen\n"
+	        "                      font produces weird results.\n"
+	        "  program[=<value>]:  Display the name of a running program instead of\n"
+	        "                      'titlebarbar_text'. The <value> can be one of:\n"
+	        "                        none:     Do not display program name.\n"
+	        "                        name:     Program name, with file extension (default).\n"
+	        "                        path:     Name, extension, and full absolute path.\n"
+	        "                        segment:  Displays program memory segment name.\n"
+	        "                      or you can provide a custom name in one of the forms:\n"
+	        "                        =\"Game\", ='Game', =(Game), =<Game> or =[Game]\n"
+	        "                      Note: With some software (like Windows 3.1x in enhanced\n"
+	        "                      mode) it is impossible to recognize the full program name\n"
+	        "                      or path; in such case 'segment' value is used instead.\n"
+	        "  dosbox:             DOSBox emulator name.\n"
+	        "  version[=<value>]:  Display DOSBox version information after emulator name.\n"
```

Comment:
Just `Display DOSBox version information.` (If `dosbox` is not set and `program` is something else than `none`, the version will come after the DOS program/path/segment name.)

Reply:
OK, changed. This was a remain of the old configuration system.